src/components/LoadingPlaceholder.vue

```diff
@@ -98,8 +68,10 @@ export default {
 		placeholderData() {
 			const data = []
 			for (let i = 0; i < this.count; i++) {
-				// generate random widths
-				data.push('width: ' + (Math.floor(Math.random() * 20) + 30) + '%')
+				data.push({
+					amount: this.type === 'messages' ? 4 : this.type === 'conversations' ? 2 : 1,
+					width: (Math.floor(Math.random() * 40) + 30) + '%', // generate random widths for last line
```

Comment:
We don't need the random width for participants so let's not exhaust it (e.g:  in case we have 1000 participants) . I suggest to utilize the data.width to add the desired width for the participant.

```suggestion
					width: this.type === 'participants' ? '60%' : (Math.floor(Math.random() * 40) + 30) + '%', // generate random widths for last line
```

Reply:
It doesn't make sense to show 1000 participants placeholders at once, as they can't fit on screen, but yes, this will be a proper solution =)

Reply:
yes , corrected